src/core/dev_api/tensor_conversion_util.hpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/runtime/host_tensor.hpp"
+#include "openvino/runtime/tensor.hpp"
+
+namespace ov {
+namespace util {
+
+/**
+ * @brief Make temporay tensor from host tensor pointer.
+ *
+ * @param t  Input tensor for conversion.
+ * @return ov::Tensor which points to host tensor data. Can return not allocated or special dynamic depends on input
+ * tensor state.
+ */
+OPENVINO_API Tensor make_tmp_tensor(const ngraph::HostTensorPtr& t);
```

Comment:
Maybe `wrap_tensor`?
Also I propose to deprecate these methods.

Reply:
Rename and marked deprecated.